androbd/build.gradle

```diff
@@ -44,4 +44,5 @@ dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     implementation 'com.github.mik3y:usb-serial-for-android:3.3.0'
     implementation 'com.github.anastr:speedviewlib:1.6.1'
+    implementation 'androidx.core:core:1.12.0'
```

Comment:
Have you checked the increase of size by adding androidx.core?

Reply:
No, just tried to get it to work